salt/haproxy/haproxy.cfg.jinja

```diff
@@ -24,9 +24,10 @@ listen kubernetes-master
         mode tcp
         default-server inter 10s fall 3
         balance roundrobin
+        option httpchk GET /healthz
 
 {%- for minion_id, nodename in salt['mine.get']('roles:kube-master', 'nodename', 'grain').items() %}
-        server master-{{ minion_id }} {{ nodename }}:{{ pillar['api']['int_ssl_port'] }} check
+        server master-{{ minion_id }}:{{ pillar['api']['int_ssl_port'] }} {{ nodename }}:{{ pillar['api']['int_ssl_port'] }} check-ssl verify none
```

Comment:
Should this not be `check ssl verify none` instead of `check-ssl verify none` ?

And - any reason we're adding the port the backend name? That name is just an arbitrary label haproxy asks for - it could be anything, so long as it's unique - for consistency - we should either leave the port off, or add it to all the names for all the services fronted by haproxy.

Reply:
According to HAProxy's documentation I see `check-ssl`: https://www.haproxy.com/documentation/aloha/7-0/traffic-management/lb-layer7/health-checks/.

I will drop the port on the name, yeah.

Reply:
Yes, `check-ssl` is valid for our version of Haproxy v1.6
https://cbonte.github.io/haproxy-dconv/configuration-1.6.html#5.2-check-ssl